src/containers/Nav.jsx

```diff
@@ -5,14 +5,15 @@ import styles from '../styles/nav'
 import React, { Component } from 'react'
 import classNames from 'classnames'
 import { translate } from '../lib/I18n'
-import { Link } from 'react-router'
+
 import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
 
 import Spinner from '../components/Spinner'
-import { FILES_CONTEXT, TRASH_CONTEXT } from '../constants/config'
+import { ROOT_DIR_ID, TRASH_DIR_ID } from '../constants/config'
 import { openFolder } from '../actions'
 
-class ActiveLink extends Component {
+class Link extends Component {
```

Comment:
Isn't it dangerous to use a `<Link />` component that differs to the `react-router` `<Link />` component?